Tests/UpstreamIntegrationTests/ConfigOverrideTests.swift

```diff
@@ -0,0 +1,323 @@
+//
+// Copyright 2023 Adobe. All rights reserved.
+// This file is licensed to you under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may obtain a copy
+// of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under
+// the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+// OF ANY KIND, either express or implied. See the License for the specific language
+// governing permissions and limitations under the License.
+//
+
+@testable import AEPCore
+import AEPEdge
+import AEPEdgeIdentity
+import AEPServices
+import Foundation
+import XCTest
+
+/// Performs validation on integration with the Edge Network upstream service
+class ConfigOverrideTests: TestBase {
+    private var edgeEnvironment: EdgeEnvironment = getEdgeEnvironment()
+    private var edgeLocationHint: EdgeLocationHint? = getLocationHint()
+
+    private var networkService: RealNetworkService = RealNetworkService()
+
+    let LOG_SOURCE = "ConfigOverrideTests"
+
+    let asyncTimeout: TimeInterval = 10
+
+    let url = "https://obumobile5.data.adobedc.net/ee/v1/interact"
+
+    // Run before each test case
+    override func setUp() {
+        ServiceProvider.shared.networkService = networkService
+
+        super.setUp()
+
+        continueAfterFailure = true
+        TestBase.debugEnabled = true
+
+        let waitForRegistration = CountDownLatch(1)
+        MobileCore.setLogLevel(.trace)
+
+        print("Using Edge Network environment: \(edgeEnvironment)")
```

Comment:
nit: can be removed from all tests, you already print this line inside getEdgeEnvironment